core/src/Streamly/Internal/Data/Array.hs

```diff
@@ -539,3 +551,85 @@ deserialize arr@(Array {..}) = unsafeInlineIO $ do
         Serialize.deserializeAt arrStart arrContents (arrStart + lenArr)
     assertM(off == arrStart + lenArr)
     pure val
+
+-------------------------------------------------------------------------------
+-- Streams of Arrays
+-------------------------------------------------------------------------------
+
+-- TODO: efficiently compare two streams of arrays. Two streams can have chunks
+-- of different sizes, we can handle that in the stream comparison abstraction.
+-- This could be useful e.g. to fast compare whether two files differ.
```

Comment:
Can you open as issue with respect to this?

Reply:
Not worth.